lib/std/build.zig

```diff
@@ -3492,3 +3492,58 @@ test "LibExeObjStep.addPackage" {
     const dupe = exe.packages.items[0];
     try std.testing.expectEqualStrings(pkg_top.name, dupe.name);
 }
+
+test "build_runner issue 10381" {
```

Comment:
Please specify what is being tested rather than referencing a github issue number